vcl/CMakeLists.txt

```diff
@@ -91,8 +91,8 @@ if("${VXL_INSTALL_INCLUDE_DIR}" STREQUAL "include/vxl")
   set(_compilers_destination vcl/compilers)
   set(_compiler_detection_destination vcl)
 else()
-  set(_compilers_destination compilers)
-  set(_compiler_detection_destination .)
+  set(_compilers_destination ${VXL_INSTALL_INCLUDE_DIR}/compilers)
```

Comment:
Doesn't this precisely contradict c98b2c2c83818bc8300e8a1d3c1a7c3f481b553f?

Reply:
Does e335538d81d4ad24347307bbd5f352cf8325f146 make this OK?

Reply:
> Doesn't this precisely contradict c98b2c2?

Yes, but that patch did not result in the correct install location.

> Does e335538 make this OK?

No, because the detection headers are still installed.